kubernetes/samples/charts/util/setupLoadBalancer.sh

```diff
@@ -3,15 +3,22 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
 # This script is to create or delete Ingress controllers. 
-# We support two ingress controllers: Traefik and Voyager.
+# Currently the script supports two ingress controllers: Traefik and Voyager.
+# Usage $0 create|delete traefik|voyager traefik_version|voyager_version
 
 UTILDIR="$(dirname "$(readlink -f "$0")")"
```

Comment:
Is there a test that covers this script?  I wonder because (a) no integration test that calls this script appears to have changed in this pull, and (b) the namespace doesn't seem to be configurable where-as I thought the integration tests are expected to use a dynamic namespace for its load balancers.   If not, perhaps file a JIRA to have this addressed?

Reply:
As of now (in develop) we do not have Traefik integration test, as Traefik 1.x has been unstable in Helm3.  We have automated tests for Ngnix and Voyager.  Currently Sankar is working on adding Traefik LoadBalancer. In integration we build the ingress object from the scratch using kubectl API not thru setupLodBalacer.sh scripts. I see  2 references to the script in our doc 
  kubernetes/samples/charts/voyager/README.md
  kubernetes/samples/charts/traefik/README.md 

We can add a extra parameter to add load balancer namespace with suitable default in another JIRA

Reply:
The MII sample uses Traefik, and there's an IT test for it - but the MII IT test doesn't use this particular script. Also, I think Sankar has a pull out for Traefik IT testing - but, again, I doubt it uses this script. We should have IT tests for this sample script (and similarly other shell/yaml).

Reply:
We try to avoid usage of script(s)/artifacts  in integration test. That was the decision taken when we start writing JUNIT5 tests.  May be Mii Sample is a candidate to use the script. Sample using sample script.

Reply:
> We try to avoid usage of script(s)/artifacts in integration test. That was the decision taken when we start writing JUNIT5 tests.

I think it's reasonable to point out that the samples should be easily tested, but, apart from the MII sample, aren't. There's no requirement that the samples should form the core of the test framework, and the fact that they aren't the core should not prevent them from being tested IMO.

Regardless, I've voted up with the assumption that all doc, shell, and YAML changes have been at least manually tested...

Reply:
If so can you approve the PR.